### PR TITLE
Depends update + deprecate

### PR DESCRIPTION
**Pull request by ascott1**
This updates dependencies and adds a deprecation notice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,4 +50,4 @@
     "request": "~2.34.0",
```
Comment (Scotchester) [inline package.json]: Did the Snyk updater not run on dev dependencies? Request is the one we're getting alerted about.

Comment (ascott1) [inline package.json]: Ahh, mayhaps

Comment (ascott1) [inline package.json]: Updated

**Review by Scotchester — APPROVED** [review]
👍

**Comment by Scotchester** [comment]
@ascott1 You want to handle the release or should I?

**Pull request merged**